src/NServiceBus.AcceptanceTests/Sagas/When_saga_message_goes_through_delayed_retries.cs

```diff
@@ -31,9 +31,9 @@ public class Context : ScenarioContext
             public int NumberOfTimesInvoked { get; set; }
         }
 
-        public class SagaMessageThroughDelayedRetryEndpoint : EndpointConfigurationBuilder
+        public class Saga09Endpoint : EndpointConfigurationBuilder
```

Comment:
This gets useless to the point where we can actually random guid name the endpoints. I really dislike where this is moving.